docs/source/module_configuration/staticsite.rst

```diff
@@ -43,71 +305,62 @@ Incorporating CloudFront with a static site is a common best practice, however,
 if you are working on a development project it may benefit you to add the
 `staticsite_cf_disable` environment parameter set to `true`.
 
-.. _staticsite-config-options:
+`Auth@Edge`
+-----------
 
-Example of all Static Site Options
-----------------------------------
+Authorization, the function of specifying access rights to resources, is often
+required to help protect restricted content in web applications. In a
+`blog post <https://aws.amazon.com/blogs/networking-and-content-delivery/authorizationedge-how-to-use-lambdaedge-and-json-web-tokens-to-enhance-web-application-security/>`_
+written by Alex Tomic and Cameron Worrell an in-depth solution is described
+on how to accomplish this with CloudFront, `Lambda@Edge` functions, and a
+Cognito User Pool. A similar solution has been integrated directly into
+Runway, via the StaticSite module, to allow easy and repeatable idempotent
+launching of private static sites.
 
-Most of these options are not required, but are listed here for reference::
+The following diagram depicts a high-level overview of this solution.
 
-    deployments:
-      - modules:
-          - name: conduitsite  # defaults to path; used in stack names & ssm parameter
-            path: web
-            class_path: runway.module.staticsite.StaticSite
-            parameters:
-              # The only required parameter value is namespace
-              namespace: contoso-${env DEPLOY_ENVIRONMENT}
-              staticsite_acmcert_arn: arn:aws:acm:us-east-1:123456789012:certificate/...
+.. image:: ../images/staticsite/auth_at_edge/flow_diagram.png
```

Comment:
If we're going to include a diagram here, needs to be our own design. I'm fine just skipping it though, with the understanding that anyone going deeper can reference the same info on one of the AWS examples.

Reply:
Already did that sir, it's our graphic